extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -215,30 +215,30 @@ private function calculateAndSaveScore() {
 			// notes for later refactoring
 			// what do I need here?
 			// - number of points based on level - give level, get points
-			foreach ( $this->mBadges as $badge_type_id => $badge_laps ) {
+			foreach( $this->mBadges as $badge_type_id => $badge_laps ) {
 
 				$badgeType = AchConfig::getInstance()->getBadgeType( $badge_type_id );
 
-				if ( $badgeType == BADGE_TYPE_NOTINTRACKSTATIC ) {
+				if( $badgeType == BADGE_TYPE_NOTINTRACKSTATIC ) {
 
-					$score += AchConfig::getInstance()->getLevelScore( $notInTrackStatic[$badge_type_id]['level'] ) * ( ( AchConfig::getInstance()->isInfinite( $badge_type_id ) ) ? $badge_laps : 1 );
+					$score += AchConfig::getInstance()->getLevelScore( $notInTrackStatic[ $badge_type_id ][ 'level' ] ) * ( ( AchConfig::getInstance()->isInfinite( $badge_type_id ) ) ? $badge_laps : 1 );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "level" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)